website/docs/r/notification_destination.html.markdown

```diff
@@ -243,39 +243,40 @@ resource "newrelic_notification_destination" "foo" {
 
 In order to create a Slack destination, you have to grant our application access to your workspace. This process is [based on OAuth](https://api.slack.com/authentication/oauth-v2) and can only be done through a browser.
 As a result, you cannot set up a Slack destination purely with Terraform code.
-You can either create a Slack destination in the interface and [import](#import) it to Terraform, or use destination data source.
-
+You can either use [the destination data source](https://registry.terraform.io/providers/newrelic/newrelic/latest/docs/data-sources/notification_destination),
+or reference destination id directly. Destination ID which can be found through [the destination data source](https://registry.terraform.io/providers/newrelic/newrelic/latest/docs/data-sources/notification_destination),
+our GraphQL API, or using the user interface.
```

Comment:
```suggestion
However, if you would like to use Slack-based destinations with other resources in the New Relic Terraform Provider, the [data source `newrelic_notification_destination`](https://registry.terraform.io/providers/newrelic/newrelic/latest/docs/data-sources/notification_destination),
may be used to fetch the ID of the destination; alternatively, you might want to source the ID of the destination from  NerdGraph, or from the New Relic One UI.
```

(just trying to make this paragraph look better - please let me know if it sounds okay, too :))

Reply:
There's no real need to use the slack-based destination on TF, you can just use the id itself, which is already needed to get this resource or data source in the first place.
So this update might make it a bit confusing...

Reply:
You're right, but I just wanted to rephrase your sentence, just to make it slightly more better grammatically (and not add anything else to the sentence); so I think we might want to have this change (maybe? Let me know what you think :)